packages/applicant/src/components/partials/Header.module.scss

```diff
@@ -1,6 +1,19 @@
 @import '~govuk-frontend/govuk/base';
 @import '~govuk-frontend/govuk/core/global-styles.scss';
 
-.float-right{
-    float: right;
-  }
\ No newline at end of file
+.float-right {
+  float: right;
+}
+
+.d-none-mobile {
+  display: block;
+  white-space: pre;
+  position: absolute;
+  right: 0;
+}
```

Comment:
probs want to rename this since it does more than just set the display eg `.super-admin-link`